tools/update_versions.py

```diff
@@ -118,8 +132,8 @@ def _bump_release(self, pkg, change_log_header=None, user_name=None,
             new_commit_id = new_source["commit_id"]
             LOG.info("Updating package %s from %s to %s" % (
                 self.pkg.name, old_commit_id, new_commit_id))
-            change_log_lines += _get_git_log(
-                new_source["repo"], old_commit_id)
+            change_log_lines += _get_git_last_commit_log(
+                new_source["repo"], new_commit_id)
```

Comment:
You could use the [commit function](http://gitpython.readthedocs.io/en/1.0.1/reference.html#git.repo.base.Repo.commit) and [summary attribute](http://gitpython.readthedocs.io/en/1.0.1/reference.html#git.objects.commit.Commit.summary) to make this much simpler and remove the functions `_get_git_last_commit_log` and `_get_git_log`:
```
SHORT_HEX_SHA_LENGTH = 7
commit = new_source["repo"].commit(new_commit_id)
change_log_lines += "Updating to commit %s: %s" % (
    commit.hexsha[:SHORT_HEX_SHA_LENGTH], commit.summary))